modules/imaging_browser/php/mrinavigation.class.inc

```diff
@@ -46,11 +46,8 @@ class MRINavigation
     {
         $this->_sessionID           = $sessionID;
         $this->_FilteredSessionList = $_SESSION['State']
-            ->getProperty('mriSessionsListed');
+            ->getProperty('mriSessionsListed') ?? [];
 
-        if (!is_array($this->_FilteredSessionList)) {
```

Comment:
I think this needs to stay. It's theoretically possible for the property to be a non-null non-array value, and in that case we still want it to be treated as the empty array.